tests/providers/amazon/aws/operators/test_s3_bucket_tagging.py

```diff
@@ -0,0 +1,120 @@
+            task_id=TASK_ID,
+            bucket_name=BUCKET_NAME,
+        )
+
+    @mock_s3
+    @mock.patch.object(S3Hook, "get_bucket_tagging")
+    @mock.patch.object(S3Hook, "check_for_bucket")
+    def test_execute_if_bucket_exist(self, mock_check_for_bucket, get_bucket_tagging):
+        mock_check_for_bucket.return_value = True
+        # execute s3 get bucket tagging operator
+        self.get_bucket_tagging_operator.execute({})
+        mock_check_for_bucket.assert_called_once_with(BUCKET_NAME)
+        get_bucket_tagging.assert_called_once_with(BUCKET_NAME)
+
+    @mock_s3
+    @mock.patch.object(S3Hook, "get_bucket_tagging")
+    @mock.patch.object(S3Hook, "check_for_bucket")
+    def test_execute_if_not_bucket_exist(self, mock_check_for_bucket, get_bucket_tagging):
+        mock_check_for_bucket.return_value = False
+        # execute s3 get bucket tagging operator
+        self.get_bucket_tagging_operator.execute({})
+        mock_check_for_bucket.assert_called_once_with(BUCKET_NAME)
+        get_bucket_tagging.assert_not_called()
+
+
+class TestS3PutBucketTaggingOperator(unittest.TestCase):
+    def setUp(self):
+        self.put_bucket_tagging_operator = S3PutBucketTaggingOperator(
+            task_id=TASK_ID,
+            tag_set=TAG_SET,
+            bucket_name=BUCKET_NAME,
+        )
+
+    @mock_s3
+    @mock.patch.object(S3Hook, "put_bucket_tagging")
+    @mock.patch.object(S3Hook, "check_for_bucket")
+    def test_execute_if_bucket_exist(self, mock_check_for_bucket, put_bucket_tagging):
+        mock_check_for_bucket.return_value = True
+        # execute s3 put bucket tagging operator
+        self.put_bucket_tagging_operator.execute({})
+        mock_check_for_bucket.assert_called_once_with(BUCKET_NAME)
+        put_bucket_tagging.assert_called_once_with(
+            key=None, value=None, tag_set=TAG_SET, bucket_name=BUCKET_NAME
+        )
+
+    @mock_s3
+    @mock.patch.object(S3Hook, "put_bucket_tagging")
+    @mock.patch.object(S3Hook, "check_for_bucket")
+    def test_execute_if_not_bucket_exist(self, mock_check_for_bucket, put_bucket_tagging):
+        mock_check_for_bucket.return_value = False
+        # execute s3 put bucket tagging operator
+        self.put_bucket_tagging_operator.execute({})
+        mock_check_for_bucket.assert_called_once_with(BUCKET_NAME)
+        put_bucket_tagging.assert_not_called()
+
+
+class TestS3DeleteBucketTaggingOperator(unittest.TestCase):
+    def setUp(self):
+        self.delete_bucket_tagging_operator = S3DeleteBucketTaggingOperator(
+            task_id=TASK_ID,
+            bucket_name=BUCKET_NAME,
+        )
+
+    @mock_s3
+    @mock.patch.object(S3Hook, "delete_bucket_tagging")
+    @mock.patch.object(S3Hook, "check_for_bucket")
+    def test_execute_if_bucket_exist(self, mock_check_for_bucket, delete_bucket_tagging):
+        mock_check_for_bucket.return_value = True
+        # execute s3 get bucket tagging operator
```

Comment:
nit: Looks like this comment was copy/pasted from the Get test class, s/get/delete/g
Same in the method below.

Reply:
Indeed.   Corrected, thanks.